correspondence/rebuttal-1.md

```diff
@@ -33,12 +33,16 @@ Manuscript: "Future challenges for Research Software Engineering"
 
 ### Suggestions
 
+As suggested also by other referees, we made the workshop notes available at zenodo. A link has been made part of the paper.
+
```

Comment:
I don't think this sentence needs to be repeated here, especially not as the first one in the section.

```suggestion
```